docs/monitors/telegraf-win_perf_counters.md

```diff
@@ -39,22 +39,22 @@ Monitor Type: `telegraf/win_perf_counters`
 | Config option | Required | Type | Description |
 | --- | --- | --- | --- |
 | `objects` | no | `list of object (see below)` |  |
-| `counterRefreshInterval` | no | `integer` | number of nanoseconds that wildcards in counter paths should be expanded and how often to refresh counters from configuration (**default:** `60`) |
-| `useWildCardExpansion` | no | `bool` | if `true`, instance indexes will be included in instance names, and wildcards will be expanded and localized (if applicable).  If `false`, non partial wildcards will be expanded and instance names will not include instance indexes. (**default:** `false`) |
-| `printValid` | no | `bool` | print out the configurations that match available performance counters (**default:** `false`) |
+| `counterRefreshInterval` | no | `int64` | Number of nanoseconds that wildcards in counter paths should be expanded and how often to refresh counters from configuration (**default:** `5s`) |
```

Comment:
I just realized the type generated for `time.Duration` is misleading.  It is technically an int64 underneath but is parsed as a string, which is what should go here.  We'll just merge this as is and fix the doc generation later.